packages/atlas/src/views/viewer/VideoView/VideoView.tsx

```diff
@@ -135,6 +139,7 @@ export const VideoView: FC = () => {
   const categoryId = video?.category?.id
   const numberOfLikes = video?.reactions.filter(({ reaction }) => reaction === 'LIKE').length
   const numberOfDislikes = video?.reactions.filter(({ reaction }) => reaction === 'UNLIKE').length
+  const videoIsBlocked = !!(id && filteredVideoIds.includes(id))
```

Comment:
This was mentioned in the original issue (https://github.com/Joystream/atlas/issues/3213), this new screen should be shown whenever the video in unavailable, for whatever reason. With your current code this new screen will be shown only if the video ID is blocked, but following cases would be ignored:
- Video is outside of app's category list
- Video was marked as censored by curators
- Video just doesn't exist
This new view should be used in place of `EmptyFallback` used previously